[debug] [2020-04-02T07:43:39.545Z] ----------------------------------------------------------------------
[debug] [2020-04-02T07:43:39.547Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\songs\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js init
[debug] [2020-04-02T07:43:39.547Z] CLI Version:   8.0.0
[debug] [2020-04-02T07:43:39.547Z] Platform:      win32
[debug] [2020-04-02T07:43:39.547Z] Node Version:  v12.13.1
[debug] [2020-04-02T07:43:39.548Z] Time:          Thu Apr 02 2020 16:43:39 GMT+0900 (GMT+09:00)
[debug] [2020-04-02T07:43:39.548Z] ----------------------------------------------------------------------
[debug] [2020-04-02T07:43:39.548Z] 
[debug] [2020-04-02T07:43:39.555Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-04-02T07:43:39.574Z] Error: Could not load the default credentials. Browse to https://cloud.google.com/docs/authentication/getting-started for more information.
    at GoogleAuth.getApplicationDefaultAsync (C:\Users\songs\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:160:19)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async GoogleAuth.getClient (C:\Users\songs\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:502:17)
    at async GoogleAuth.getAccessToken (C:\Users\songs\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\google-auth-library\build\src\auth\googleauth.js:524:24)
[error] 
[error] Error: An unexpected error has occurred.
